cardano-chain-gen/src/Cardano/Mock/Forging/Tx/Babbage.hs

```diff
@@ -643,13 +637,13 @@ mkFullTx n m sta = do
         [unregisteredKeyHash !! 1, unregisteredKeyHash !! 2]
 
     certs =
-      [ DCertDeleg $ RegKey $ Prelude.head unregisteredStakeCredentials
-      , DCertPool $ RegPool poolParams0
-      , DCertPool $ RegPool poolParams1
-      , DCertPool $ RetirePool (Prelude.head unregisteredPools) (EpochNo 0)
-      , DCertDeleg $ DeRegKey $ unregisteredStakeCredentials !! 2
-      , DCertDeleg $ Delegate $ Delegation (unregisteredStakeCredentials !! 1) (unregisteredPools !! 2)
-      , DCertMir $
+      [ ShelleyTxCertDelegCert $ ShelleyRegCert $ Prelude.head unregisteredStakeCredentials
```

Comment:
Just checking I assume there will never be a scenario where the list is empty in the tests, since there is the use of `head`?